pages/faqs.md

```diff
@@ -53,16 +53,56 @@ If your command-line client was installed manually, please check the path of the
 ### I get the "16-bit MS-DOS Subsystem" error dialog after upgrading on Windows. What should I do?
 Prior to version 3.0.5 of the Exercism command-line client, there was a bug in the upgrade command that would replace the command-line client binary file with a single text file causing the "16-bit MS-DOS Subsystem" error. To resolve this issue remove the corrupt binary and reinstall the command-line client using the [interactive walkthrough](https://exercism.io/cli-walkthrough).
 
-### What if my issue is not listed on this page?
-  If your problem hasn't been resolved or reported, then [create an issue](https://github.com/exercism/cli/issues/new) with the following information:
+## Mentored Mode
 
-  1. The output of the `exercism troubleshoot` command.
-  1. Details on the issue you are experiencing.
-  1. Instructions on how to reproduce the issue.
+### My queue hasn't updated in a while - is there something wrong?
+While wait times can be longer than normal for a number of reasons (number of mentors, time of year), it can also appear that the queue is not updating. This is not unusual and you should see movement fairly soon. If you your wait is far longer than the average, refer to [Opening an Issue](#opening-an-issue) below.
 
-## Exercism Team FAQs
+### How many mentors does my track have? I only see a handful on the website.
+The website only shows mentors who have provided bio information for the website, not the actual number of mentors who are actively reviewing solutions. Rest assured, there are mentors working through their queues so hang in there!
 
-### What happened to my Team from the old site?
+### I submitted the wrong solution / file. What can I do?
+There isn't a way to delete submissions so just submit the right file and it will show up as a new iteration in the queue. Feel free to leave a note for your mentor with the original submission clarifying what happened.
+
+### How can I report abuse or examples of bad mentoring?
+Please check our [Code of Conduct](https://exercism.io/code-of-conduct) for more information about our expectations of conduct. If you would like to report something, please reach out to us at [abuse@exercism.io](mailto:abuse@exercism.io?subject=%5BCoC%5D]) and we will try to fix or resolve the issue respecting both you and your privacy.
+
+## Improving Exercism
+
+### This is great! How do I get involved?
+There are a few different ways - becoming a mentor, managing a language track or reporting (or addressing!) issues on GitHub. You can see more on the [How to Contribute page](https://exercism.io/contribute). Version 3 of Exercism is also being built right now and needs help from track maintainers. The [v3 repository](https://github.com/exercism/v3) has more information.

```

Comment:
Just a minor nit: people can also contribute to v3 while not being a maintainer:

```suggestion
There are a few different ways - becoming a mentor, managing a language track or reporting (or addressing!) issues on GitHub. You can see more on the [How to Contribute page](https://exercism.io/contribute). Version 3 of Exercism is also being built right now and needs lots of help. The [v3 repository](https://github.com/exercism/v3) has more information.
```